csi/csi.go

```diff
@@ -44,7 +44,14 @@ import (
 	volumedrivers "github.com/libopenstorage/openstorage/volume/drivers"
 )
 
-var clogger *logrus.Logger
+var (
+	clogger *logrus.Logger
+)
+
+const (
+	connCleanupInterval = 15 * time.Minute
+	connIdleConnLength  = 1 * time.Hour
```

Comment:
May be we can drop this to 30 mins. So the max time a connection will live is <= 45mins

Reply:
Sounds good :+1: